### ISSUE TITLE
fp16 and learning rate

### ISSUE THREAD

**Issue by LiweiPeng** [pytorch/fairseq#334]
I am using fairseq on machine translation models. Compared with fp32, if using fp16, does the learning rate parameters need to be changed? Thanks,

**Comment by myleott** [comment]
Typically we use the same learning rate with FP16. Are you observing different training behavior between FP16 and FP32 with the same learning rate?
